package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pim-community-standard",
-  "version": "3.1.0",
+  "version": "3.2.0",
```

Comment:
Are you sure about this ? Shouldn't we do this change after we checkout the 3.2 branch ?